### PR TITLE
Machine updates

### PR DESCRIPTION
**Pull request by NinaWie**
Change 6144 "Mall of America - I Love Minnesota Store":
	 Paywall from: False to: True
	Address from: 372 West Market - Level 3, Bloomington to: 372 West Market - Level 1, Bloomington
	 Location from: 44.8545, -93.2433 to: 44.8537, -93.2408

### CODE REVIEW

**Comment by jannisborn** [comment]
No website updates today!

**Comment by jannisborn** [comment]
2024-04-08 22:30:02 INFO ======Location differ joblog from 2024-04-08 22:30:02=======
2024-04-08 23:08:53 INFO Capitol Visitor's Center got permanently removed
2024-04-08 23:12:06 INFO Location Texas (26/125): Changes in 1/297 machines found.
2024-04-08 23:12:35 INFO 41/128: Found machine to be added: Covington Visitor Center
2024-04-08 23:13:36 INFO Location Georgia (27/125): Changes in 1/128 machines found.
2024-04-08 23:17:51 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-04-08 23:47:16 INFO 99/376: Found machine to be added: Schloss Benrath
2024-04-08 23:51:23 INFO Location Germany (73/125): Changes in 1/376 machines found.
2024-04-09 00:08:04 INFO Electronica & Souvenir Shop is currently unavailable
2024-04-09 00:08:25 INFO Location Spain (112/125): Changes in 1/167 machines found.
2024-04-09 00:10:51 INFO 
 Result: 4 changes, 2 new machines found and 1 machines retired
2024-04-09 00:26:15 ERROR Found 4 problems that require manual intervention
2024-04-09 00:26:15 INFO ======Location differ completed at 2024-04-09 00:26:15=======
2024-04-09 00:26:16 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-04-09 22:30:01 INFO ======Location differ joblog from 2024-04-09 22:30:01=======
2024-04-09 22:54:38 INFO 26/105: Found machine to be added: Mall of America - The Neighborhood
2024-04-09 22:55:34 INFO Location Minnesota (16/125): Changes in 1/105 machines found.
2024-04-09 23:14:50 INFO New York New York Hotel & Casino - Hershey's Chocolate World is available again
2024-04-09 23:15:24 INFO Location Nevada (31/125): Changes in 1/204 machines found.
2024-04-09 23:15:37 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-04-10 00:06:26 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-04-10 00:06:52 ERROR Our machine Capitol Reef National Park Visitor Center in Utah from Device shown as available but http://209.221.138.252/Details.aspx?location=380412 responds Internal Server Error (500)
2024-04-10 00:20:51 ERROR Found 4 problems that require manual intervention
2024-04-10 00:20:51 INFO ======Location differ completed at 2024-04-10 00:20:51=======
2024-04-10 00:20:51 INFO Detected change in server_locations.json - push to github

**Comment by jannisborn** [comment]
2024-04-10 22:30:02 INFO ======Location differ joblog from 2024-04-10 22:30:02=======
2024-04-10 23:15:50 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-04-10 23:39:34 INFO Fusiliers Museum of Northumberland is currently unavailable
2024-04-10 23:40:39 INFO Location England (69/125): Changes in 1/436 machines found.
2024-04-10 23:58:58 INFO Castelo de S. Jorge is currently unavailable
2024-04-10 23:59:19 INFO Location Portugal (101/125): Changes in 1/71 machines found.
2024-04-11 00:03:09 INFO 111/168: Found machine to be added: Castro de Santa Tecla
2024-04-11 00:03:55 INFO Location Spain (112/125): Changes in 1/168 machines found.
2024-04-11 00:06:15 INFO 
 Result: 3 changes, 1 new machines found and 2 machines retired
2024-04-11 00:07:54 ERROR Our machine Pena Palace in Portugal from Device shown as available but http://209.221.138.252/Details.aspx?location=336844 responds Internal Server Error (500)
2024-04-11 00:19:51 ERROR Our machine Schmitten Groer Feldberg (Taunus) in Germany from Server shown as available but http://elongated-coin.de/phpBB3/viewtopic.php?t=1767 responds Bad Gateway (502)
2024-04-11 00:20:40 ERROR Found 4 problems that require manual intervention
2024-04-11 00:20:40 INFO ======Location differ completed at 2024-04-11 00:20:40=======
2024-04-11 00:20:40 INFO Detected change in server_locations.json - push to github

File: data/server_locations.json [hunk 1]
```diff
@@ -31172,13 +31173,13 @@
                 "area": "Germany",
                 "address": "Großer Feldberg 1, 61389 Schmitten, Germany",
                 "status": "unvisited",
-                "external_url": "http://elongated-coin.de/phpBB3/viewtopic.php?t=1767",
+                "external_url": "null",
```
Comment (jannisborn) [inline data/server_locations.json:31176]:
```suggestion
                "external_url": "http://elongated-coin.de/phpBB3/viewtopic.php?t=1767",
```

**Comment by jannisborn** [comment]
2024-04-11 22:30:02 INFO ======Location differ joblog from 2024-04-11 22:30:02=======
2024-04-11 23:15:09 ERROR Geolocation failed for: Unknown Location 	 sub: , Arco
2024-04-11 23:33:13 INFO 1/9: Found machine to be added: Orlova Chuka Cave
2024-04-11 23:33:16 INFO 3/9: Found machine to be added: EcoMuseum
2024-04-11 23:33:21 INFO Location Bulgaria (60/125): Changes in 2/9 machines found.
2024-04-12 00:05:25 INFO 
 Result: 2 changes, 2 new machines found and 0 machines retired
2024-04-12 00:19:45 ERROR Found 4 problems that require manual intervention
2024-04-12 00:19:45 INFO ======Location differ completed at 2024-04-12 00:19:45=======
2024-04-12 00:19:46 INFO Detected change in server_locations.json - push to github

**Comment by NinaWie** [comment]
New machine 7135 named Fort Morgan State Historic Site submitted.

**Comment by NinaWie** [comment]
New machine 7136 named Test machine submitted.

File: data/server_locations.json (hunk 1 above)
Comment (NinaWie) [reply data/server_locations.json:31176]: Somehow I cannot commit your change with the link. Everytime I try to click on "Commit change", it's saying "This diff has recently been updated. Refresh and try again". Refreshing the page and trying againd did not help. Can you commit yourself?

**Review by jannisborn — APPROVED** [review]

**Pull request merged**